client/src/components/Input/LocationSearch.js

```diff
@@ -26,15 +31,15 @@ const LocationSearch = () => {
       <WhiteSpace />
       <StyledSearchBar
```

Comment:
`prefix={<SvgIcon />}` doesn't work?

Btw, `SvgIcon` is a component we're using for SVG icons. You can check out the implementation or reach out to @nadinejuraschek who created it

Reply:
That didn't work.

Reply:
This is the actual folder that they used here. It doesn't look like we can change the icon. https://mobile.ant.design/components/search-bar/#components-search-bar-demo-basic